### ISSUE TITLE
Rename `webservice` storageType data models to `generic`

### ISSUE THREAD

**Issue by asishallab** [Zendro-dev/graphql-server-model-codegen#130]
Once the reopened issue #120 is done, before it is merged into master rename the storageType `webservice` to the _new_ storageType `generic`.

This affects the code generators on the level of processing the provided input (data model definitions) and possibly some templates where the storageType is checked.

Update integration tests accordingly. Patches, data model definitions and the such. Run the whole unit and integration test suites before marking this as ready for revision.

Also update the existing documentation, both on sciencedb.github.com and in our Readme's.
